tests/components/unifi/test_sensor.py

```diff
@@ -122,9 +143,18 @@ async def test_sensors(hass):
     wireless_client_tx = hass.states.get("sensor.wireless_client_name_tx")
     assert wireless_client_tx is None
 
+    wired_client_uptime = hass.states.get("sensor.wired_client_name_uptime")
+    assert wired_client_uptime is None
+
+    wireless_client_uptime = hass.states.get("sensor.wireless_client_name_uptime")
+    assert wireless_client_uptime is None
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6184085)